tests/storages_tests/test_storages.py

```diff
@@ -139,6 +139,22 @@ def test_create_new_study_with_name(storage_mode, cache_mode):
         assert study_name == storage.get_study_name_from_id(study_id)
 
 
```

Comment:
Please add a test like the following scenario for RDBStorage:
1. Create multiple studies
2. Remove one of them
3. Assert the deleted study was actually removed from the study list